processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -74,6 +77,76 @@
 {
   private volatile DateTime maxIngestedEventTime;
 
+  private static final Map<String, ValueType> TYPE_MAP = ImmutableMap.<String, ValueType>builder()
+      .put("Long[]", ValueType.LONG)
```

Comment:
Just curious, do we have any parser in Druid currently that returns `[]` type for multi-valued dimension ?